.gitlab-ci-prod.yml

```diff
@@ -39,3 +40,13 @@ deploy:
   only:
     - prod
   when: manual
+
+test:
+  stage: test
+  script:
+    -make smoketest-prod
+  environment:
```

Comment:
The `environment` block should be omitted for non-deployment stages

Reply:
addressed